docs_app/content/guide/v6/pipeable-operators.md

```diff
@@ -0,0 +1,245 @@
+# Pipeable Operators
+
+Starting in version 5.5 we have shipped "pipeable operators", which can be accessed in `rxjs/operators` (notice the pluralized "operators"). These are meant to be a better approach for pulling in just the operators you need than the "patch" operators found in `rxjs-compat` package.
+
+**NOTE**: Using `rxjs` or `rxjs/operators` without making changes to your build process can result in larger bundles. See [Known Issues](#known-issues) section below.
+
+**Renamed Operators**
+
+Due to having operators available independent of an Observable, operator names cannot conflict with JavaScript keyword restrictions. Therefore the names of the pipeable version of some operators have changed. These operators are:
+
+1. `do` -> `tap`
+2. `catch` -> `catchError`
+3. `switch` -> `switchAll`
+4. `finally` -> `finalize`
+
+The `let` operator is now part of `Observable` as `pipe` and cannot be imported.
+
+`source$.let(myOperator) -> source$.pipe(myOperator)`
+
+See "[Build Your Own Operators](#build-your-own-operators-easily)" below.
+
+The former `toPromise()` "operator" has been removed
+because an operator returns an `Observable`,
+not a `Promise`.
+There is now an `Observable.toPromise()`instance method.
+
+## Why?
+
+Problems with the patched operators for dot-chaining are:
+
+1. Any library that imports a patch operator will augment the `Observable.prototype` for all consumers of that library, creating blind dependencies. If the library removes their usage, they unknowingly break everyone else. With pipeables, you have to import the operators you need into each file you use them in.
+
+2. Operators patched directly onto the prototype are not "tree-shakeable" by tools like rollup or webpack. Pipeable operators will be as they are just functions pulled in from modules directly.
+
+3. Unused operators that are being imported in apps cannot be detected reliably by any sort of build tooling or lint rule. That means that you might import `scan`, but stop using it, and it's still being added to your output bundle. With pipeable operators, if you're not using it, a lint rule can pick it up for you.
+
+4. Functional composition is awesome. Building your own custom operators becomes much, much easier, and now they work and look just like all other operators from rxjs. You don't need to extend Observable or override `lift` anymore.
+
+## What?
+
+What is a pipeable operator? Simply put, a function that can be used with the current `let` operator. It used to be the origin of the name ("lettable"), but that was confusing and we call them "pipeable" now because they're intended to be used with the `pipe` utility. A pipeable operator is basically any function that returns a function with the signature: `<T, R>(source: Observable<T>) => Observable<R>`.
```

Comment:
```suggestion
What is a pipeable operator? Simply put, a function that can be used with the current `let` operator. It used to be the origin of the name ("lettable"), but that was confusing so we now call them "pipeable" because they're intended to be used with the `pipe` utility. A pipeable operator is basically any function that returns a function with the signature: `<T, R>(source: Observable<T>) => Observable<R>`.
```